charter.md

```diff
@@ -0,0 +1,355 @@
+
+This working group aims to provide:
+
+- a set suite of realistic sample applications, which employ observability tools to demonstrate how practitioners can gain insight and understanding.
+- actionable, best-practice guidance on how to apply observability to real-life workloads. Observed workloads will be CNCF Projects.
+
+_A distinct section requiring intentional navigation by the User shall be provided for open source projects which are not CNCF Projects if this opportunity arises._
+
+## Interested Parties
+
+- Matt Young ([halcyondude@gmail.com](mailto:halcyondude@gmail.com))
+- Ivan Santos ([pragmaticivan@gmail.com](mailto:pragmaticivan@gmail.com), EverQuote)
+- Ken Finnigan ([ken@kenfinnigan.me](mailto:ken@kenfinnigan.me), Workday)
+- Michael Hausenblas ([mh9@o11y.engineering](mailto:mh9@o11y.engineering), AWS)
+- Ryan Perry ([rperry2174@gmail.com](mailto:rperry2174@gmail.com), Pyroscope)
+- Daniel Khan ([daniel@khan.io](mailto:daniel@khan.io), Elastic)
+- Severin Neumann ([severin.neumann@altmuehlnet.de](mailto:severin.neumann@altmuehlnet.de), Cisco)
+
+## Discussion(s) thus far:
+
+- CNCF: Chris A, Bill M
+- Scott Rigby (GitOps WG)
+- Bartek
+- Alolita
+- Friends / fellow Observability geeks :)
+- TAG Observability!
+- Crossplane, ACK, Pixie, Linkerd, ...
+
+## Prior discussion takeaways (as of 2021-11-22)
+
+### CNCF
+
+- Will purchase the domains if we make this real. has acquired the following domains, the CNCF would assume ownership and make us admin/root.
+  - **observe-k8s.io**
+  - **observe-k8s.dev**
+  - observe-k8s.com
+  - observe-k8s.info
+  - github.com/observe-k8s.
+- CNCF has an [Infrastructure Lab](https://github.com/cncf/cluster), or we could host on public cloud (perhaps easier than futzing with metal)
+- Guidance from Chris Aniszczyk (CNCF CTO): Use the working model that the GitOps WG is taking with “OpenGitOps” - in terms of governance. Spend time here to make it sustainable by a community of maintainers / TAG so it can have legs. Similar feedback from Scott Rigby.
+
+### GitOps WG
+
+- Demonstrate GitOps good practices.
+- PR: GitOps WG Charter to include TAG Observability as a related / interested group
+- Open Standard for reporting on execution of GitOps workflows.
+
+##
+
+## Brainstorm: Vision / Ideas
+
+### Working Group Requirements for Launch
+
+```
+    TAGs may choose to spawn focused and time-limited working groups to achieve some of their responsibilities (for example, to produce a specific educational white paper, or portfolio gap analysis report). Working groups should have a clearly documented charter, timeline (typically a few quarters at most), and set of deliverables. Once the timeline has elapsed, or the deliverables delivered, the working group dissolves, or is explicitly re-chartered.
+```
+
+    (source: https://github.com/cncf/toc/blob/main/tags/cncf-tags.md#responsibilities--empowerment-of-tags)
+
+1. Charter / Description of WG
```

Comment:
Looks like the nested lists got messed up in the generation.

Reply:
aye, will fix